Controller/NotifyHandler.php

```diff
@@ -292,4 +295,57 @@ private function _shouldUseTransactionID($sTransactionId, $sPaymentId)
     {
         return is_null($sTransactionId) || $sPaymentId === CreditCardPaymentMethod::getName(true);
     }
+
+    /**
+     * Handle credit card
+     *
+     * @param SuccessResponse $oResponse
+     * @param BackendService  $oBackendService
+     * @param Order           $oOrder
+     *
+     * @return void
+     *
+     * @since 1.3.0
+     */
+    private static function _handleCreditCard($oResponse, $oBackendService, $oOrder)
+    {
+        $oLogger = Registry::getLogger();
+
+        if (!self::_shouldSaveCard($oResponse, $oOrder)) {
+            return;
+        }
+
+        try {
+            $aTransaction = $oBackendService->getTransactionByTransactionId(
+                $oResponse->getTransactionId(),
+                CreditCardTransaction::NAME
+            );
+
+            $oVault = new Vault();
+            $oVault->saveCard($oResponse, $aTransaction['payment']['card'], $oOrder);
+        } catch (DatabaseConnectionException $oExc) {
+            $oLogger->error("Cannot save card", [$oExc]);
+        } catch (DatabaseErrorException $oExc) {
+            $oLogger->error("Cannot save card", [$oExc]);
+        }
+    }
+
+    /**
+     * Checks if the credit card credentials should be saved.
+     * For that the user needs an account, it has to be a credit card payment and the user must had checked the save
```

Comment:
typo: must ~~had~~ **have** checked

Please also add a '.' at the end of the sentence.